spaghetto/src/Spago/Log.purs

```diff
@@ -123,14 +129,76 @@ data OutputFormat a
   | OutputTable { titles :: Array String, rows :: Array (Array String) }
   | OutputLines (Array String)
 
+foreign import supportsColor :: Effect Boolean
+
+indent2 :: forall a. Doc a -> Doc a
+indent2 = Log.indent <<< Log.indent
+
 output :: forall a m. MonadEffect m => OutputFormat a -> m Unit
 output format = Console.log case format of
   OutputJson codec json -> Json.printJson codec json
   OutputLines lines -> String.joinWith "\n" lines
   -- https://github.com/natefaubion/purescript-dodo-printer/blob/master/test/snapshots/DodoBox.purs
-  OutputTable { titles, rows } -> "TODO: Unimplemented"
-
-foreign import supportsColor :: Effect Boolean
-
-indent2 :: forall a. Doc a -> Doc a
-indent2 = Log.indent <<< Log.indent
+  OutputTable { titles, rows } ->
+    Log.print Log.plainText (Log.twoSpaces)
+      $ Box.toDoc
+      $ logTable { headers: textBox <$> titles, rows: (map textBox) <$> rows }
+
+textBox :: forall a. String -> DocBox a
+textBox = print Box.docBox twoSpaces <<< Dodo.text
+
+-- TODO: Change visual appearance
+logTable
```

Comment:
Table implementation was missing. I took the implementation mostly from the comment. There is still room for improvement, but looks okay. Don't want to blow up the PR with more  stuff, but needed some output, to be able to use `OutputTable`.
https://github.com/natefaubion/purescript-dodo-printer/blob/master/test/snapshots/DodoBox.purs

Reply:
## Example Output

![image](https://user-images.githubusercontent.com/13085980/219890526-710a78b8-92bb-4f6b-aa4d-8c43918016f0.png)

Reply:
Yes! Beautiful! I meant to implement this with Dodo, but got busy with other things, so I just left the link in there. Very happy to see this implemented! 👏 
Now we can use this to fix the output for the `spago registry` commands, but not in this PR 😄